src/main/java/com/princexml/wrapper/Prince.java

```diff
@@ -434,7 +435,8 @@ private boolean readMessagesFromStderr(Process process) throws IOException {
      * Rather than retrieving documents beginning with {@code URL}, get them
      * from the local directory {@code DIR}. This method can be called more than
      * once to add multiple remappings.
-     * @param remap The remapping "URL=DIR".
+     * @param remap The remapping string in the format "URL=DIR"
+     *              (e.g. "http://example/=../images").
      */
     public void addRemap(String remap) {
```

Comment:
Perhaps `addRemap` should take _two_ arguments. :thinking:

Reply:
Oh duh :man_facepalming:, that would save having to explain it in the comments.